### ISSUE TITLE
Release 2.60 just broke my app with "There is no function named "setsubtract"" error

### ISSUE THREAD

**Issue by ramonpm** [terraform-aws-modules/terraform-aws-vpc#531]
```
Error: Call to unknown function

  on .terraform/modules/vpc/main.tf line 539, in resource "aws_default_network_acl" "this":
 539:   subnet_ids = setsubtract(
 540: 
 541: 
...

There is no function named "setsubtract".
```
I'm rolling back to 2.59, that's working fine.

**Comment by antonbabenko** [comment]
`setsubtract` is necessary, it was released in [v0.12.21](https://github.com/hashicorp/terraform/blob/v0.12/CHANGELOG.md#01221-february-19-2020)

**Comment by antonbabenko** [comment]
Please upgrade or use the previous version of Terraform.
